src/js/store.js

```diff
@@ -1,14 +1,14 @@
 /*global process*/
 /*esfmt-ignore-start*/
 import notifications from './reducers/notifications';
-import points        from './reducers/points';
-import tracks        from './reducers/tracks';
-import settings      from './reducers/settings';
-import network       from './reducers/network';
-import map           from './reducers/map';
-import filters       from './reducers/filter';
-import account       from './reducers/account';
-import drawer        from './reducers/drawer';
```

Comment:
Undo the changes in this file... This is an issue with windows esfmt-ignore not working.